src/backend/api/Fusion.Resources.Api/Controllers/Requests/DepartmentRequestsController.cs

```diff
@@ -52,6 +55,71 @@ public async Task<ActionResult<ApiCollection<ApiResourceAllocationRequest>>> Get
             return new ApiCollection<ApiResourceAllocationRequest>(apiModel);
         }
 
+
+        [EmulatedUserSupport]
+        [HttpOptions("/departments/{departmentString}/resources/requests")]
+        public async Task<ActionResult> OptionsDepartmentRequests([FromRoute] OrgUnitIdentifier departmentString)
```

Comment:
Added options endpoint to take advantage of emulate support in the future as well as testing current change.